include/Music/Display.h

```diff
@@ -0,0 +1,47 @@
+//
+// Created by adven on 2024/3/27.
+//
+
+#ifndef FUCK_PTSD_DISPLAY_H
+#define FUCK_PTSD_DISPLAY_H
+
+#include <iostream>
+
+#include "GameElement.h"
+#include "SpriteSheet.hpp"
```

Comment:
I believe that using Util/SpriteSheet.hpp would be a better solution. This is because SpriteSheet has a built-in timer, and since this is a rhythm game, I require an accurate trigger to ensure that it won’t display incorrect animations. 
Incorrect tempo animations can significantly degrade the rhythm game experience.

Reply:
I don't think that the use the `Util/SpriteSheet.hpp` can make tempo correct.

Reply:
and if the tempo show on frames early or later, we can use the value `offset` to fix it

Reply:
i will change here
```cpp
void Display::Update() {

    if (m_CurrentMusicTime >= m_triggerTime - m_range &&
        m_CurrentMusicTime <= m_triggerTime + m_range) {
        m_TempoHeartImage->Play();
    }
}
```

Reply:
and Util/SpriteSheet.hpp has not timer that you say.
<img width="1470" alt="image" src="https://github.com/NTUT-FUCK-PTSD/Fuck-PTSD/assets/98307453/c4337838-ae72-481e-a2fb-9b37131fac4a">

Reply:
SpriteSheet has a built-in timer
But Util/SpriteSheet.hpp has not a built-in timer

Reply:
因為他不是基於時間的，所以我認為更適合這個應用場景